charts/oauth/templates/deployment.yaml

```diff
@@ -39,6 +39,9 @@ spec:
       - image: {{ .Values.dex.image.repository }}:{{ .Values.dex.image.tag }}
         name: dex
         command: ["/usr/local/bin/dex", "serve", "/etc/dex/cfg/config.yaml"]
+        {{- if .Values.dex.env }}
+        env: {{ .Values.dex.env | toYaml | trim | nindent 8}}
```

Comment:
I think you can avoid one pipe:
```
{{ toYaml .Values.dex.env | trim | nindent 8 }}
```

Reply:
Personally, I prefer to read pipelines from left to right, which is kind of circumvented by having the first function on the left side of the input value.
Nevertheless: that thing is small enough and easy to understand in both variants. I'll add your recommendation and fix that missing space before the closing brackets as well :smiley:

Reply:
Also done!